core/src/main/java/org/lflang/federated/launcher/FedLauncherGenerator.java

```diff
@@ -424,56 +434,54 @@ private String getRemoteLaunchCode(
    * @param remoteBase The root directory on the remote machine.
    * @param federate The federate to distribute.
    */
-  private String getDistCode(Path remoteBase, FederateInstance federate) {
+  private String getDistCode(Path remoteBase, String name, String user, String host) {
```

Comment:
The function is not changed. Only its arguments have changed so that it can be used to generate distribution code for the RTI as well